app/graphql/concerns/validate_submission_gradable.rb

```diff
@@ -1,43 +1,59 @@
 module ValidateSubmissionGradable
   extend ActiveSupport::Concern
 
-  class OwnersShouldBeActive < GraphQL::Schema::Validator
+  class SubmissionShouldBeGradable < GraphQL::Schema::Validator
     def validate(_object, _context, value)
-      submission = TimelineEvent.find(value[:submission_id])
+      @error = nil
+      @submission = TimelineEvent.find_by(id: value[:submission_id])
 
-      #days since submission
+      submission_exists?(value[:submission_id]) && owners_active? &&
+        submission_live?
+      @error
+    end
+
+    def submission_exists?(submission_id)
+      return true if @submission.present?
+
+      @error =
+        I18n.t(
+          "mutations.create_grading.submission_missing_error",
+          submission_id: submission_id
+        )
+
+      false
+    end
```

Comment:
How does this work 🤔 

If the submission doesn't exist the request may not even reach the validator as the authorization should fail; 🤔 (We can't figure out resource school without a resource)